docs/en/deployment.md

```diff
@@ -55,6 +60,39 @@ This automated AWS CloudFormation template deploys the solution in the AWS Cloud
 
 You can view the status of the stack in the AWS CloudFormation Console in the **Status** column. You should receive a CREATE_COMPLETE status in approximately 10 minutes.
 
+### Deploy solution from source code
+
+Deploy solution to your AWS account from CDK.
+#### Prerequisites
+
+* Make sure you have AWS CLI and AWS CDK install in your workspace
```

Comment:
We should not require installing `cdk` separately,  the cdk is managed by project dependencies.

The `awscli` also is not required for cdk app. The prerequisites should be `node.js` LTS, such as 12.x and 14.x.

Reply:
changed